### DISCUSSION TITLE
how do I use in the browser?

### DISCUSSION THREAD

**Discussion by ralyodio** [js-fns/date-fns#2193] [Q&A]
I'm using vanilla js and loaded the lib from a cdn.

That works fine, but how do I use the functions?

` 12     const ago = formatRelative(item.createdAt);`

`formatRelative` is undefined.

**Comment by ralyodio** [comment]
So i noticed it's `dateFns.format()` is the only format function available. How do I get the others?

**Reply by tan75** [reply]
hi, 

the _**formatRelative**_ function takes 2 parameters - the date to format and the date to compare with.
[https://date-fns.org/v2.17.0/docs/formatRelative](url)

Here is an example:

```
const dateFns = require('date-fns');

const fr = dateFns.formatRelative(
  new Date(2014, 1, 12, 11, 30, 30),
  new Date(2014, 1, 11, 11, 30, 30)
);

console.log(fr); // => tomorrow at 11:30 AM
```

**Reply by ralyodio** [reply]
using in browser sir

**Reply by tan75** [reply]
@chovy 
can you try to use this one?
[https://www.skypack.dev/view/date-fns](url)

```
import('https://cdn.skypack.dev/date-fns').then((dateFns) =>
  console.log(
    dateFns.formatRelative(
      new Date(2014, 1, 12, 11, 30, 30),
      new Date(2014, 1, 11, 11, 30, 30)
    )
  )
);
```

![image](https://user-images.githubusercontent.com/8555279/107364713-b9a25780-6ad3-11eb-86ab-cfcba01e874c.png)

**Reply by ralyodio** [reply]
what is `import`?

**Reply by ralyodio** [reply]
i get a cors error on that.

**Reply by tan75** [reply]
hi @chovy 

> I'm using vanilla js and loaded the lib from a cdn. - 

What version of date-fns did you load? 
formatRelative is available only in date-fns  2.x


The code below works ok for me.

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Document</title>
  </head>
  <body></body>
  <script>
    import('https://cdn.skypack.dev/date-fns').then((dateFns) => {
      console.log(dateFns);
      console.log(
        dateFns.formatRelative(
          new Date(2014, 1, 12, 11, 30, 30),
          new Date(2014, 1, 11, 11, 30, 30)
        )
      );
    });
  </script>
</html>

```
